### ISSUE TITLE
Error v1.1.0

### ISSUE THREAD

**Issue by TiRajo** [xhlove/WVGuesser#3]
File "E:\WVGuesser-masterv3\WVGuesser-master\wvguesser\mainv2.py", line 20, in <module>
    client.connect(('127.0.0.1', port))
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

**Comment by xbstory** [comment]
have no idea for this ……

**Comment by anaveem** [comment]
I get the same error. And the compiled exe copy doesn't work either. It just runs and exits abruptly. The only forked copy I found to work is this: https://github.com/1453374029/WVGuesser

Could be due to users using different versions of Python and its libraries. I would encourage anyone who is developing this further to list the version of Python and required libraries. If it helps, I'm using a freshly installed copy of Python.

```
C:\anon\desktop\xhlove WVGuesser-master>python -m wvguesser.mainv2
Traceback (most recent call last):
  File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\anon\desktop\xhlove WVGuesser-master\wvguesser\mainv2.py", line 26, in <module>
    client.connect(('127.0.0.1', port))
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it
```

**Comment by PuMa2006** [comment]
> I get the same error. And the compiled exe copy doesn't work either. It just runs and exits abruptly. The only forked copy I found to work is this: https://github.com/1453374029/WVGuesser
> 
> Could be due to users using different versions of Python and its libraries. I would encourage anyone who is developing this further to list the version of Python and required libraries. If it helps, I'm using a freshly installed copy of Python.
> 
> ```
> C:\anon\desktop\xhlove WVGuesser-master>python -m wvguesser.mainv2
> Traceback (most recent call last):
>   File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
>     exec(code, run_globals)
>   File "C:\anon\desktop\xhlove WVGuesser-master\wvguesser\mainv2.py", line 26, in <module>
>     client.connect(('127.0.0.1', port))
> ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it
> ```

Yup. Same problem here. But compiled (1.2.0) works

**Comment by TiRajo** [comment]
> > I get the same error. And the compiled exe copy doesn't work either. It just runs and exits abruptly. The only forked copy I found to work is this: https://github.com/1453374029/WVGuesser
> > Could be due to users using different versions of Python and its libraries. I would encourage anyone who is developing this further to list the version of Python and required libraries. If it helps, I'm using a freshly installed copy of Python.
> > ```
> > C:\anon\desktop\xhlove WVGuesser-master>python -m wvguesser.mainv2
> > Traceback (most recent call last):
> >   File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
> >     return _run_code(code, main_globals, None,
> >   File "C:\Users\anon\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
> >     exec(code, run_globals)
> >   File "C:\anon\desktop\xhlove WVGuesser-master\wvguesser\mainv2.py", line 26, in <module>
> >     client.connect(('127.0.0.1', port))
> > ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it
> > ```
> 
> Yup. Same problem here. But compiled (1.2.0) works

Got fixed its due to space in Folder name Please Remove the Space In Folder Name ! Even in v1.2.0 face connection refuse issues so rename the folder without space.

**Comment by anaveem** [comment]
@TiRAjo, can you be more specific? Which space, in which folder name? thanks

**Comment by TiRajo** [comment]
> @TiRajo, can you be more specific? Which space, in which folder name? thanks

C:\anon\desktop\xhlove(**_remove this space_**)WVGuesser-master>python -m wvguesser.mainv2 - ! use this files in folder without special characters so it will be fine in running

**Comment by anaveem** [comment]
Ah, ok, thanks, yes that was the problem. I thought you were referring to spaces in the program folders, because users shouldn't have to avoid using spaces in their folder names. I believe what's happening is that mainv2.py is calling the program path without quotes ' ', so anyone who placed the program in a folder with spaces will get that error. @xhlove it would be great if you could fix that in your code for the next commit. Thanks

**Comment by xbstory** [comment]
its fixed in v1.3.0.
